native/acter/api.rsh

```diff
@@ -239,6 +239,12 @@ object UserProfile {
 }
 
 object RoomProfile {
+    /// get room id
+    fn room_id() -> RoomId;
```

Comment:
What's the use of duplicating this information methods when we only seems to use string cases ? I don't recall having ```RoomId``` object being of any benefit over front-end...

Reply:
technically, it would be a better if we were to pass only `RoomId` types in and out of the API, but that isn't really feasible until we've moved to uniffi. I have removed these from the api.rsh but left the actual `fn`s in place for future usage.